web/src/main/java/com/futurewei/alcor/web/entity/node/NodeInfo.java

```diff
@@ -57,27 +57,33 @@
     @JsonProperty("group_topic")
     private String groupTopic;
 
+    @JsonProperty("ncm_uri")
+    private String ncm_uri;
```

Comment:
@pkommoju If url is part of NodeInfo, in the case that we NCM service url is updated, we will need to modify all impacted nodes. 

Another option is to create a new KV cache, ncm_id => ncm_info. Think about this option.

Reply:
I am looking into it.

Reply:
I think the deciding factor could be frequency of updating URI vs frequecny of having to join. Say, a given URI_NCM_one is on NCM_one, which is managing node_000 through node_999 (1000 nodes). If URI_one of NCM_one is changed to URI_NCM_one_new, then 1000 nodeinfo will have to be updated every time URI_NCM_xyz changes.
On the otherhand, DPM/NCM will have to take the node_info.ncm_id and join (query ncm_id => ncm_info) to get the NCM URI.